lib/Sema/SemaBounds.cpp

```diff
@@ -524,25 +636,25 @@ namespace {
           MemberExpr *M = dyn_cast<MemberExpr>(E);
           if (!M) {
             llvm_unreachable("unexpected cast failure");
-            return CreateBoundsNone();
+            return CreateBoundsInferenceError();
           }
 
           FieldDecl *F = dyn_cast<FieldDecl>(M->getMemberDecl());
           if (!F)
-            return CreateBoundsNone();
+            return CreateBoundsInferenceError();
 
           BoundsExpr *B = F->getBoundsExpr();
           if (!B || B->isNone() || B->isInteropTypeAnnotation())
-            return CreateBoundsNone();
+            return CreateBoundsInferenceError();
```

Comment:
Then the first two are `CreateBoundsUnknown()` and the third is `CreateBoundsAllowedButNotComputed()` right?